<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="VideoFrame">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>301</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="VideoLayout"/>
    </widget>
   </widget>
   <widget class="QPushButton" name="WordGenerator">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Nowe słowo</string>
    </property>
   </widget>
   <widget class="QFrame" name="TipFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="TipLayout"/>
    </widget>
   </widget>
   <widget class="QComboBox" name="CategoryPicker">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Cechy charakteru</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cechy fizyczne</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dodatkowe</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Elementy ubioru</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Emocje</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Geografia Polski</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Geografia świata</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Imiona</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jedzenie</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kolory</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Meble</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Przedmioty codziennego użytku</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="WordDisplay" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>371</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>475</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
